src/plugins_datastore.h

```diff
@@ -221,6 +221,17 @@ typedef int (*srds_access_get)(const struct lys_module *mod, sr_datastore_t ds,
  */
 typedef int (*srds_access_check)(const struct lys_module *mod, sr_datastore_t ds, int *read, int *write);
 
+/**
+ * @brief Get the time when the datastore data of the module were last modified.
+ *
+ * @param[in] mod Specific module.
+ * @param[in] ds Specific datastore.
+ * @param[out] mtime Time of last modification.
```

Comment:
Mention that `0`s are expected if it is unknown (such as when the file does not exist).

Reply:
Yes, that ought to be documented.